daemon/src/taker_cfd.rs

```diff
@@ -326,6 +329,51 @@ impl<O, M, W> Actor<O, M, W> {
         }
         Ok(())
     }
+
+    async fn handle_propose_roll_over(&mut self, proposal: RollOverProposal) -> Result<()> {
+        self.current_pending_proposals.insert(
+            proposal.order_id,
+            UpdateCfdProposal::RollOverProposal {
+                proposal: proposal.clone(),
+                direction: SettlementKind::Outgoing,
+            },
+        );
+        self.send_pending_update_proposals()?;
+
+        self.send_to_maker
+            .do_send(wire::TakerToMaker::ProposeRollOver {
+                order_id: proposal.order_id,
+                timestamp: proposal.timestamp,
+            })?;
+        Ok(())
+    }
+}
+
+impl<O: 'static, M: 'static, W: 'static> Actor<O, M, W> {
+    async fn handle_auto_rollover(&mut self, ctx: &mut Context<Self>) -> Result<()> {
+        tracing::debug!("Auto rollover");
+
+        let mut conn = self.db.acquire().await?;
+        let cfds = load_all_cfds(&mut conn).await?;
+
+        // cleanup all pending proposals because auto-rollover will create a new one
+        self.current_pending_proposals.clear();
```

Comment:
Note for tomorrow Daniel: Is this actually needed given that we filter?

Reply:
Keeping this statement here means that we clean up rollovers every 5 minutes. I think this is fine. We could also just do it in case we have a proposal. I don't think it should do any harm to do this here - what do you think @thomaseizinger ?